plugins/reportal/src/azkaban/reportal/util/ReportalUtil.java

```diff
@@ -57,4 +62,63 @@ public static IStreamProvider getStreamProvider(String fileSystem) {
 
     return sortedNodes;
   }
+
+  /**
+   * Get runtime variables to be set in unscheduled mode of execution.
+   * Returns empty list, if no runtime variable is found
+   *
+   * @param variables
+   * @return
+   */
+  public static List<Variable> getRunTimeVariables(
+    Collection<Variable> variables) {
+    List<Variable> runtimeVariables =
+      ReportalUtil.getVariablesByRegex(variables, "(?!(^"
```

Comment:
I think   .*Reportal.REPORTAL_CONFIG_PREFIX.+   or even the prefix match will be good enough,  as you are not "renaming" the variable name so I think it doesn't matter for the match result here .